### DISCUSSION TITLE
Switch Client to different sync server

### DISCUSSION THREAD

**Discussion by riegera2412** [TriliumNext/Trilium#4193] [General]
Heya,

is there a method to reset the client (Win 10x64 most recent version on Git Hub) and then connect it to a different docker instance I just set up for work notes exclusively? 

I didn't find any related discussions or issues or a reset button in the client. Simply switching the URL of the sync server instance didn't work either (I was desperate so I just tried knowing it probably won't work.). Lastly, I just deleted the client directory on my pc and then downloaded a fresh copy from Git Hub. However, after starting the client it still has the old settings, which must be stored somewhere in the depths of Windows. 

If any of you have an idea what to do to switch my client to a different server, I'd appreciate any advice or pointers you can give me.

Regards,
André

**Comment by Nriver** [comment]
You can either delete the old data, it usually here `C:\Users\yourname\AppData\Roaming\trilium-data`

Or, you can use `trilium-portable.bat` to run in a portable mode, the data will be in the same directory of your trilium.exe

**Reply by riegera2412** [reply]
Worked like a charm and was simple enough! 
Also, great advice using the trilium-portable.bat! I plan to use both servers on the same PC. By working with two separate client directories and using the trilium-portable.bat in both, I should be able to use both servers, if I understand correctly. Trying this at home on Friday, when I am working at home again.

Many thanks @Nriver !
